analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -88,39 +87,17 @@
 # | `-1` | 2269 | 3.59 TB |
 #
 # The input files are all in the 1–2 GB range.
-#
-# Some files are also rucio-accessible (with ATLAS credentials):
-#
-# | dataset | number of files | total size |
-# | --- | --- | --- |
-# | `user.ivukotic:user.ivukotic.ttbar__nominal` | 7066 | 1.46 TB |
-# | `user.ivukotic:user.ivukotic.ttbar__scaledown` | 902 | 209 GB |
-# | `user.ivukotic:user.ivukotic.ttbar__scaleup` | 917 | 191 GB |
-# | `user.ivukotic:user.ivukotic.ttbar__ME_var` | 438 | 103 GB |
-# | `user.ivukotic:user.ivukotic.ttbar__PS_var` | 443 | 100 GB |
-# | `user.ivukotic:user.ivukotic.single_top_s_chan__nominal` | 114 | 11 GB |
-# | `user.ivukotic:user.ivukotic.single_top_t_chan__nominal` | 2506 | 392 GB |
-# | `user.ivukotic:user.ivukotic.single_top_tW__nominal` | 50 | 9 GB |
-# | `user.ivukotic:user.ivukotic.wjets__nominal` | 10199 | 1.13 TB |
-# | total | 22635 | 3.61 TB |
-#
-# The difference in total file size is presumably due to the different storages, which report slightly different sizes.
-#
-# When setting the `PIPELINE` variable below to `"servicex_databinder"`, the `N_FILES_MAX_PER_SAMPLE` variable is ignored and all files are processed.
 
 # %%
 ### GLOBAL CONFIGURATION
 
 # input files per process, set to e.g. 10 (smaller number = faster)
 N_FILES_MAX_PER_SAMPLE = 5
 
-# pipeline to use:
-# - "coffea" for pure coffea setup
-# - "servicex_processor" for coffea with ServiceX processor
-# - "servicex_databinder" for downloading query output and subsequent standalone coffea
-PIPELINE = "coffea"
+# enable ServiceX
+USE_SERVICEX = False
```

Comment:
This is pretty minor, but perhaps group this together with the cache setting since they're both about ServiceX.

Reply:
Addressed in [9587d6c](https://github.com/iris-hep/analysis-grand-challenge/pull/103/commits/9587d6c0f7766965c0f95282c5a78fbcf1c14001)